standard/sections/annex_c_examples.adoc

```diff
@@ -8,15 +8,19 @@
 .Example: Notification from Environment and Climate Change Canada, Meteorological Service of Canada, Global Discovery Catalogue Service, concerning a WCMP2 record from Météo-France (Toulouse)
 [source,text]
 ----
-monitor/a/wis2/ca-eccc-msc-global-discovery-catalogue/fr-meteofrance
+monitor/a/wis2/ca-eccc-msc-global-discovery-catalogue
 ----
 
+Here, the `subject` is found in the WMEM (`fr-meteofrance`).
```

Comment:
Here, the `source` is found in the WMEM (`ca-eccc-msc-global-discovery-catalogue`).